store/src/main/java/com/nytimes/android/external/store3/pipeline/PipelineStore.kt

```diff
@@ -45,13 +36,13 @@ interface PipelineStore<Key, Output> {
 fun <Key, Output> PipelineStore<Key, Output>.open(): Store<Output, Key> {
     val self = this
     return object : Store<Output, Key> {
-        override suspend fun get(key: Key) = self.get(
-            StoreRequest.cached(key, refresh = false)
-        )!!
+        override suspend fun get(key: Key) = self.stream(
+                StoreRequest.cached(key, refresh = false)
+        ).take(1).toList().first()
```

Comment:
how would it be possible to not get an item. Old store through an exception on get if no items returned.  I think this should be just `first()` which throws when no items
```suggestion
        ).first()
```

Reply:
idk why but my ide does not autocomplete it, maybe we need a new version.

https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines.flow/first.html